base/docs/helpdb.jl

```diff
@@ -14567,6 +14567,21 @@ include
 doc"""
 ```rst
 ::
+           include_dependency(path::AbstractString)
+
+In a module, declare that the file specified by `path` (relative or
+absolute) is a dependency for precompilation; that is, the
+module will need to be recompiled if this file changes.
+
+This is only needed if your module depends on a file that is not
+used via `include`.   It has no effect outside of compilation.
+```
+"""
+include_dependency
```

Comment:
I don't really understand the new doc system for the manual.  @one-more-minute, can you clarify what we are supposed to do?  I'd really rather just put the documentation inline in `loading.jl`.